subt/ros/robot/launch/k2-virt.launch

```diff
@@ -0,0 +1,246 @@
+<?xml version="1.0"?>
```

Comment:
We need to discuss some strategy with the launch and json files or it is going to bury us alive.